shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -334,6 +337,18 @@ - (void)updateWindowMetrics {
   FlutterEngineSendWindowMetricsEvent(_engine, &event);
 }
 
+- (void)scheduleOnRasterTread:(dispatch_block_t)block {
+  void* copy = Block_copy((__bridge void*)block);
```

Comment:
curious, why do we need to copy the block here?

Reply:
Blocks are by default allocated on stack. It needs to be moved to heap otherwise it will get destroyed on scope exit.